esx_service/vmdk_ops.py

```diff
@@ -1474,7 +1474,7 @@ def set_vol_opts(name, tenant_name, options):
 
     vmdk_path = vmdk_utils.get_vmdk_path(path, vol_name)
 
-    logging.debug("set_vol_opts: path=%s vmdk_path=%s", path, vmdk_path)
+    logging.info("set_vol_opts: path=%s vmdk_path=%s", path, vmdk_path)
```

Comment:
Essentially  this log (and the related ones) are meant to be debug itself.

Do you mind registering the issue to revert this change ?(either due to test issue or product issue)